scripts/gen-docs.sh

```diff
@@ -0,0 +1,52 @@
+#!/bin/bash
+
+function show_help {
+    echo "Usage: $(basename "$0") [-c CHART_NAME]"
+    echo ""
+    echo "Options:"
+    echo "  -c CHART_NAME   Specify the name of the chart to generate docs for."
+    echo "  -h              Show this help message."
+    echo ""
+    echo "If CHART_NAME is not provided, the script will generate docs for all charts in the 'charts' directory."
+}
+
+CHART_NAME=""
+
+while getopts "c:h" opt; do
+    case $opt in
+        c)
+            CHART_NAME=$OPTARG
+            ;;
+        h)
+            show_help
+            exit 0
+            ;;
+        *)
+            show_help
+            exit 1
+            ;;
+    esac
+done
+
+if [[ -n "$CHART_NAME" ]]; then
+    if [[ -d "charts/$CHART_NAME" ]]; then
+        echo "Generating helm docs for chart: $CHART_NAME"
+        cd "charts/$CHART_NAME" || exit 1
+        helm-docs
```

Comment:
[question] are we expecting `helm-docs` available on machine when we're running the script

Reply:
Added validation that `helm-docs` exist before running the commands